certification/internal/policy/container/has_unique_tag.go

```diff
@@ -0,0 +1,61 @@
+package container
+
+import (
+	"strings"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	log "github.com/sirupsen/logrus"
+)
+
+// HasUniqueTagCheck evaluates the image to ensure that it has a tag other than
+// the latest tag, which is considered to be a "floating" tag and may not accurately
+// represent the same image over time.
+type HasUniqueTagCheck struct{}
+
+func (p *HasUniqueTagCheck) Validate(imgRef certification.ImageReference) (bool, error) {
+	tags, err := p.getDataToValidate(imgRef.ImageURI)
+	if err != nil {
+		return false, err
+	}
+	return p.validate(imgRef.ImageURI, tags)
+}
+
+func (p *HasUniqueTagCheck) getDataToValidate(image string) ([]string, error) {
+	skopeoEngine := SkopeoCLIEngine{}
+	runReport, err := skopeoEngine.ListTags(image)
```

Comment:
Crane has a ListTag method. I am curious to know if it is possible to return the tags for a specific image by using it: https://pkg.go.dev/github.com/google/go-containerregistry/pkg/crane#ListTags